### ISSUE TITLE
Lab2a - switching to user student in Windows

### ISSUE THREAD

**Issue by KathlynS** [WEISFall2016/MySQL-Labs#3]
2) On Windows, you can't just log in as student in a new window. When opening a new window, it automatically wants the password for root. -u student -p ict4300 did not work.  Instead, I went through the following steps: 
1)  Start>
 
2)  Right click on "MySQL 5.7 Command Line Client">More>Open file location
 
3)  Right click on "MySQL 5.7 Command Line Client" filename
 
4)  Click on Properties
 
5)  Copy what is in the 'Target' box
 
6)  Paste this command into the command line of DOS window
 
7) Change the -uroot to -ustudent
 
8)  Then it will prompt for the student password

**Comment by sammitchell410** [comment]
These are the same steps that I use to get into mysql as well. It looks like this for me:
Open a terminal window
type: "/usr/local/mysql/bin/mysql" (or I can create an alias by saying "alias mysql=/usr/local/mysql/bin/mysql")
type: "mysql -u root -p" to connect as root without going into database, or:
"mysql -u root -p mysql" to connect as root and go directly into mysql database, or:
"mysql -u student -p" to connect as student without going into a database, or:
"mysql -u student -p ict4300" to connect as student and go directly into mysql database.

After each one of these, it will prompt for password and then I will be in.
Does this look familiar to your login methods? I assume that when you copy what is in the "Target" box, it is just the path to the mysql directory. Is this correct?

What are everyone else's methods of logging in to mysql?

Sam

**Comment by KathlynS** [comment]
Sorry, this also wasn't meant to be a question, but a solution to what has to be done differently on windows than what the instructions show for mac.  In windows, you can't switch users from an open window of the mysql command line, or by selecting the mysql command line from the start menu (it pops up wanting the password for root).  I had to open a regular dos prompt and login to mysql from there with a different user.  I was listing the steps on how to do that. The tee command has to be reissued afterwards because it's a brand new session.  

Kathy

**Comment by sammitchell410** [comment]
Got it. I was just confused because the MySQL package for macs does not have a command line tool. I am doing everything (and I assume everyone else on Mac as well) from the terminal (DOS prompt on Windows). I guess what I am getting at is since the Command Line Client you are using makes it difficult to switch users, would it be easier to rewrite the instructions using solely the DOS prompt?

Sam

**Comment by KathlynS** [comment]
Hi Sam, 

That is a good suggestion, as it might be a cleaner way to do it.  However, the limitation is on Windows, not just the command line interface.  That option would work too, although it still won't let you switch users in windows with the command given "mysql -u student -p ict4300", as listed in the lab.  You still have to exit, and login the same way as the instructions in the red in the attachment below. Hopefully that made sense.  :-)

This attachment shows some suggestions I made to the lab.  What is not relevant to Windows, I changed to blue.  What should replace the material in blue for Windows, is shown in red.  I suggest we either add a Windows section, or I make a set of instructions for Windows with these small changes.  Thoughts?

[Shirey MySQL Lab2a windows updates.docx](https://github.com/WEISFall2016/MySQL-Labs/files/549430/Shirey.MySQL.Lab2a.windows.updates.docx)

**Comment by sammitchell410** [comment]
I think it looks good. My vote would be to make a new document for Lab2A specifically for windows with the changes replaced, and to eventually have a few screenshots 👍  You basically already have it changed though which is awesome. I'm not even on windows and I could follow your instructions.

Where is everyone else?! haha.

**Comment by KathlynS** [comment]
Thanks for the feedback!  Screenshots are definitely a good idea for a separate document.  Since it sounds like that's the direction we'll go, I will get those added and repost the document when it's done.

I don't know!  Where is everyone else?  haha :-)

**Comment by sammitchell410** [comment]
Awesome. I can't test it, but I would be happy to review the doc for clarity, etc. if you would like it!

**Comment by KathlynS** [comment]
Yes, thanks!  I haven't put it together yet, but will post it once I do. The first lab is the only that needs the separate document - since the rest just say to log in and don't go through the instructions again.

**Comment by KathlynS** [comment]
Here's that document with the Windows screenshots.
[Shirey MySQL Lab2a windows updates.docx](https://github.com/WEISFall2016/MySQL-Labs/files/592625/Shirey.MySQL.Lab2a.windows.updates.docx)
